doc/MIGRATION-v7.md

```diff
@@ -256,6 +258,12 @@ The OpenShift Template `objects` field (`getObjects()` / `setObjects()`) has cha
 Technically, this field can contain any kind of object, not just `HasMetadata` object instances.
 The object will always be deserialized as a `KubernetesResource` instance, but not necessarily a `HasMetadata` instance (`AnyType`, `RawExtension`, `GenericKubernetesResource`).
 
+## CRD Generator annotation processor <a href="#crd-generator-annotation-processor" id="crd-generator-annotation-processor"/>
+
+### `@PrinterColumn` format changed to enum <a href="#crd-generator-printercolumn-format" id="crd-generator-printercolumn-format"/>
+
+The type of `format` in `@PrinterColumn` has changed from string to enum `PrinterColumnFormat`.
+
```

Comment:
There's already a section covering the CRD Generator deprecation:

https://github.com/fabric8io/kubernetes-client/blob/5cab66cd6d7ca1c48ca51f3eb4fdb9c730004591/doc/MIGRATION-v7.md#L32

https://github.com/fabric8io/kubernetes-client/blob/5cab66cd6d7ca1c48ca51f3eb4fdb9c730004591/doc/MIGRATION-v7.md#L323-L328

Maybe to make it clear it's better to add this section with a `v2`?
---> `[CRD Generator annotation processor (v2)]`

Or since this was in preview mode (not sure if that was clear) just omit it completely.

Reply:
Yes, the deprecation section is fine and we can also move this noteworthy change to this.

Please keep in mind that this is for users who stay during the migration with the annotation processor solution.

This way we can show, it's only a small change required --> PrinterColumn format.

For the migration to the new tooling I plan to contribute a seperate file as it is not directly related to the v7 migration. Is this ok?

Reply:
Yes, sure